code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -35,7 +35,7 @@
 	if(reach>1)
 		. += "<span class='notice'>This weapon has a reach of [reach].</span>"
 
-	if(throwforce>force)
```

Comment:
Why? This makes every non-thrown weapon show a throwforce variable. This also is not within the scope of a typical abnormality PR

Reply:
I checked and it doesnt